### ISSUE TITLE
Restructure code

### ISSUE THREAD

**Issue by uda** [hasadna/anyway#1009]
Put views and admin views in sub-packages for easier maintenance and understanding the moving parts.

**Comment by atalyaalon** [comment]
@uda what views do you mean? DB views?

**Comment by uda** [comment]
No, the route views, each endpoint is a View, currently it uses functions, I prefer classes, but either way each group should be in a module for easier readability and maintenance.

**Comment by uda** [comment]
I propose to work on the code restructure in 2 stages:

1. Split app initialization, views and functions (helpers) that are inside `flask_app.py` into 3 separate files
2. Organize the views and functions into subject modules and common module

This will give us a better look at what we provide through the app
